CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 (none)
 
 # v4.1.0
+#### Bug fixes & Enhancements:
```

Comment:
This This section is within a session of a release already released. A PR shortly entered with the Suggested release: v5.0.0 section. Move this section there.